search/searcher/search_term_range_test.go

```diff
@@ -253,6 +253,9 @@ func TestTermRangeSearchTooManyTerms(t *testing.T) {
 		got = append(got, extId)
 		ctx.DocumentMatchPool.Put(next)
 		next, err = searcher.Next(ctx)
+		if err != nil {
+			t.Fatal(err)
+		}
```

Comment:
This one is not needed, as in the previous case, `err != nil` will immediately exit the loop, and in this case we already have the `err != nil` case afterwards.